feature/google/src/main/java/com/revenuecat/purchases/google/findDefaultOffer.kt

```diff
@@ -35,25 +36,18 @@ private val GoogleSubscriptionOption.freePricingPhase: PricingPhase?
 private val GoogleSubscriptionOption.nonFreePricingPhase: PricingPhase?
     get() = pricingPhases.firstOrNull()?.takeIf { it.price.amountMicros > 0L }
 
-private const val DAYS_IN_YEAR = 365
-private const val DAYS_IN_MONTH = 30
+private const val DAYS_IN_DAY = 1
```

Comment:
🤣 we should definitely expose this to customers. in case they don't know.

Reply:
Ha! It feels... silly but detekt made it do it because "magic numbers" 🙄